README.md

```diff
@@ -1153,6 +1153,7 @@ Update Time, five active automations, webhooks.
 
 ## Web Hosting
 
+  * [4EVERLAND](https://www.4everland.org/) — In-built development and deployment tools(e.g., visual and command line) to build, deploy and host decentralised websites/applications with consensus-driven Swarm network, free SSL, global CDN, and preview URLs. The gateway node includes dynamic DNS, Data l/0, and CertManager modules. Registered users can get free 6 GB IPFS storage, 300MB Arweve storage and 100 GB/month bandwidth.
```

Comment:
Thanks, we only list entries once so find a category that suits best.

Reply:
Thank you for your reply, if so please include us under the Iaas category, is anything new i need to do?

Reply:
Just update your PR accordingly

Reply:
It's done, thanks.